From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Piotr Szczepanik <piter75@gmail.com>
Date: Mon, 9 Dec 2019 07:39:09 +0100
Subject: [ARCHEOLOGY] Enable derivation of stable MAC address from cpuid on
 rk3399 (#1660)

> X-Git-Archeology: - Revision cd9800498d1ff5ade84f701dcd50ecfedfcc7091: https://github.com/armbian/build/commit/cd9800498d1ff5ade84f701dcd50ecfedfcc7091
> X-Git-Archeology:   Date: Mon, 09 Dec 2019 07:39:09 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Enable derivation of stable MAC address from cpuid on rk3399 (#1660)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 5a2b2c360b9c26ca4bd0d309af7cd3994fd08b7d: https://github.com/armbian/build/commit/5a2b2c360b9c26ca4bd0d309af7cd3994fd08b7d
> X-Git-Archeology:   Date: Sun, 03 May 2020 19:15:46 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Prepare rockchip64 for switch to mainline u-boot and switched ROCK Pi 4 (#1934)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 86abac1fd4e169712a44d245cf7adef4ee17c420: https://github.com/armbian/build/commit/86abac1fd4e169712a44d245cf7adef4ee17c420
> X-Git-Archeology:   Date: Tue, 26 Oct 2021 22:14:41 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Cleanup u-boot configurations for rockchip64 derivatives (#3150)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
> X-Git-Archeology:
---
 include/configs/rk3399_common.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/configs/rk3399_common.h b/include/configs/rk3399_common.h
index ed72c8bb6b..836206b4e0 100644
--- a/include/configs/rk3399_common.h
+++ b/include/configs/rk3399_common.h
@@ -17,6 +17,10 @@
 
 #define CONFIG_SYS_INIT_SP_ADDR		0x00300000
 
+#define CONFIG_MISC 1
+#define CONFIG_MISC_INIT_R 1
+#define CONFIG_ROCKCHIP_EFUSE 1
+
 #if defined(CONFIG_SPL_BUILD) && defined(CONFIG_TPL_BOOTROM_SUPPORT)
 #define CONFIG_SPL_STACK		0x00400000
 #define CONFIG_SPL_MAX_SIZE             0x40000
-- 
Armbian

